datacommons/datacommons.py

```diff
@@ -452,6 +454,52 @@ def get_states(self, country, new_col_name, max_rows=100):
 
     return pd.concat([type_row, dcid_column], ignore_index=True)
 
+
+  def get_contained_places(self,
```

Comment:
Were we originally to have arguments `place` (dcid), `contained_in_place_type` instead of `place_type`, `place_name`, and `contained_place_type`?